packages/utils/telemetry-utils/src/telemetryTypes.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Property types that can be logged.
+ *
+ * @remarks Logging entire objects is considered extremely dangerous from a telemetry point of view because people can
+ * easily add fields to objects that shouldn't be logged and not realize it's going to be logged.
+ * General best practice is to explicitly log the fields you care about from objects.
+ */
+export type TelemetryEventPropertyTypeExt = string
```

Comment:
```suggestion
export type TelemetryEventPropertyTypeExt =
```
`string` was dujplicated